src/Altinn.App.Core/Internal/App/IAppFeatures.cs

```diff
@@ -0,0 +1,14 @@
+namespace Altinn.App.Core.Internal.App
+{
+    /// <summary>
+    /// Interface reporting features needed by frontend and their status to support multiple versions of the backend
+    /// </summary>
+    public interface IAppFeatures
+    {
+        /// <summary>
+        /// Fetch frontend features that are supported by this backend
+        /// </summary>
+        /// <returns>List of frontend features enabled/disabled for this backend</returns>
+        public Task<Dictionary<string, bool>> GetEnabledFeatures();
```

Comment:
Does it get enabled features or enabled/disabled features - perhaps rename to GetFeatures?

Reply:
Agreed, will change it as soon as we conclude if we should rename features to capabilities